src/site/stages/build/process-cms-exports/schemas/raw/node-landing_page.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    changed: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string' },
+          format: { type: 'string' },
+        },
+        required: ['value', 'format'],
+      },
+    },
+    path: { $ref: 'RawPath' },
+    field_administration: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_alert: { type: 'array' },
+    field_intro_text: { $ref: 'GenericNestedString' },
+    // This is essentially a list of paragraph-link_teaser,
+    // but only containing the data in field_link
+    field_links: {
+      type: 'array',
+      // I don't know that this array will *always* have only one item,
+      // but this schema passed the content build so it seemed
+      // like a decent idea to call it out as a potential change
+      // for the templates or CMS export
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          uri: { type: 'string' },
+          title: { type: 'string' },
+          options: { type: 'array' },
+        },
+        required: ['uri', 'title', 'options'],
+      },
+    },
+    field_page_last_built: { $ref: 'GenericNestedString' },
+    field_plainlanguage_date: { type: 'array' },
```

Comment:
Did this have any examples? If we don't end up using it, I kind of don't care what `items` is...except that we _may_ later, so I'd like for it to be checked if we can.

If you only found empty arrays, it may be worth throwing in a `maxItems: 0` with a comment like `// maxItems: 0 until we know what the items look like`.